store/src/java/com/zimbra/cs/lmtpserver/LmtpAddress.java

```diff
@@ -119,7 +123,7 @@ public String getDomainPart() {
     }
 
     public boolean isValid() {
-	return mIsValid;
+		return true;
```

Comment:
why are we by-passing the validity check?

Reply:
^ @ajinkya81 did you see this question? This function is always going to return `true` now